Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -806,9 +959,20 @@ public void CreateRig()
 
         #endregion
 
+
         #region MonoBehaviour Methods
+
         private void Start()
         {
+            //override Shell Parity settings if no prefab is specified for corner handles;
+            if (scaleHandlePrefab == null)
+            {
+                closeScale = 1.8f;
+                mediumScale = 1.5f;
+                farScale = 1.0f;
+            }
+            farScale = Mathf.Max(farScale, 0.01f);
```

Comment:
Questions:
do we want to support users modifying these values during runtime? if so we wouldn't catch if someone puts a 0 there (moving it to the setter would fix that i suppose)

will we have a similar problem with the medium and close scale?

Reply:
- Removed hard-coded values. Optimized default values for the HL1 style (for the script used without prefabs)
- Removed Mathf.Max(farScale, 0.01f) line since it should allow 0 to make handle invisible
Example of FarScale = 0 / MediumScale = 1.0 / CloseScale = 1.5
![MRTK_BBHandleSteps](https://user-images.githubusercontent.com/13754172/60558277-7aa7ff80-9cfd-11e9-90e6-30d74d05a986.gif)